gammapy/datasets/map.py

```diff
@@ -1541,7 +1541,7 @@ def from_dict(cls, data, lazy=False, cache=True):
     @property
     def _counts_statistic(self):
         """Counts statistics of the dataset."""
-        return CashCountsStatistic(self.counts, self.background)
+        return CashCountsStatistic(self.counts, self.npred_background())
```

Comment:
`npred_background` includes the norm and spectrum correction, `background` does not. @registerrier which one did we intend to use for the dataset?

Reply:
Because if we keep the change above: `npred_background` in `_count_statistic`, then the background field of info_dict become `npred_background`, but we already have a field `npred_background`. 
Does that answer your question @adonath ?

Reply:
This looks for the me correct behaviour! self.npred_background() contains the adjusted bkg, and self.background only the projected IRF (or a measured OFF from a reflected...).
In the case of a 3D dataset, computing a TS with only the bkg projected IRF is a non-sense: this is the whole spirit of the FoVbackgroundmaker within our data reduction scheme. And if people this using info_table function for physics, in particular for a source detection status, this will lead to wrong result that we do not wish.
I think that this line is an important bug fix within Gammapy!